### PR TITLE
Fix travis issues by updating theano to dev

### PR DESCRIPTION
**Pull request by twiecki**
Moving back to theano dev as I think it has some fixes we need.

### CODE REVIEW

File: pymc3/distributions/multivariate.py [hunk 1]
```diff
@@ -1,14 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```
Comment (kyleam) [inline pymc3/distributions/multivariate.py]: Was this intended for examples/wishart.py instead of here?

File: pymc3/examples/wishart.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    # To be able to compare it to truth, convert precision to covariance
+    cov = pm.Deterministic('cov', T.nlinalg.matrix_inverse(prec))
+
+    lp = pm.MvNormal('likelihood', mu=mu, tau=prec, observed=data)
+
+    start = pm.find_MAP()
+    step = pm.NUTS(scaling=start)
+    trace = pm.sample(500, step)
+
```
Comment (kyleam) [inline pymc3/examples/wishart.py]: Doesn't this need to use the __main__/run setup like the other examples?

File: setup.py [hunk 3]
```diff
@@ -27,12 +27,16 @@
...
 install_reqs = ['numpy>=1.7.1', 'scipy>=0.12.0', 'matplotlib>=1.2.1',
-                'Theano>=0.7.0']
+                'Theano<=0.7dev']
 
 test_reqs = ['nose']
 if sys.version_info[0] == 2:  # py3 has mock in stdlib
     test_reqs.append('mock')
 
+## The current release of Theano does not support `as_op` decorator, so
+## we're using the developmental version from github.
```
Comment (kyleam) [inline setup.py]: I think this comment should be updated, since `as_op` is no longer the issue.

File: setup.py [hunk 4]
```diff
@@ -27,12 +27,16 @@
                'Operating System :: OS Independent']
 
 install_reqs = ['numpy>=1.7.1', 'scipy>=0.12.0', 'matplotlib>=1.2.1',
-                'Theano>=0.7.0']
+                'Theano<=0.7dev']
```
Comment (kyleam) [inline setup.py:31]: I think this need to be updated to `<=0.7.1dev`.

File: setup.py [hunk 5]
```diff
@@ -27,12 +27,16 @@
...
 
 test_reqs = ['nose']
 if sys.version_info[0] == 2:  # py3 has mock in stdlib
     test_reqs.append('mock')
 
+## The current release of Theano does not support `as_op` decorator, so
+## we're using the developmental version from github.
+dep_links = ['https://github.com/Theano/Theano/tarball/master#egg=Theano-0.7dev']
+
 if __name__ == "__main__":
```
Comment (kyleam) [inline setup.py:38]: s/0.7dev/0.7.1dev/

File: pymc3/distributions/multivariate.py (hunk 1 above)
Comment (twiecki) [inline pymc3/distributions/multivariate.py]: No, it's needed for the unicode in the new docstring.

File: pymc3/examples/wishart.py (hunk 2 above)
Comment (twiecki) [inline pymc3/examples/wishart.py]: Yes.

File: pymc3/distributions/multivariate.py (hunk 1 above)
Comment (kyleam) [inline pymc3/distributions/multivariate.py]: I was a bit inexact with my clicks.  I'm referring to the shebang.

**Comment by kyleam** [comment]
Tested the updated dependency link on my end, and it installed fine.  Merging.

**Comment by twiecki** [comment]
Thanks @kyleam

**Pull request merged**